### ISSUE TITLE
UX: Not enough contrast for link green color

### ISSUE THREAD

**Issue by merchako** [usfm-bible/docs.usfm.bible#6]
**Problem** 
The links are hard to read. They fail [WCAG AA contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) tests.

![Screenshot 2024-08-28 at 21 44 13](https://github.com/user-attachments/assets/779cde1e-2ba5-460e-9bc3-482542a49942)

Color Contrast Ratio: 2.56:1
AA Large: Fail
AA Small: Fail
AAA Large: Fail
AAA Small: Fail

Suggestion 1:
#545E17 on #FFFFFF
Color Contrast ratio 7.02:1

Suggestion 2:
#98AB29 on #161616
Color Contrast ratio 7.06:1

**Comment by klassenjm** [comment]
@merchako Considering changes in #5, do you think any further link styling changes are necessary?

**Comment by merchako** [comment]
Thank you for the color change! This is a good improvement. There's still more to do, but maybe that could be a different issue/effort. I noticed that these docs are using the Paratext 9 green. We have a new visual identity upcoming for Paratext 10 Studio. Maybe the UX team could give you colors for that. 

Is there some sort of reference doc or single page of code where colors get configured for these docs?
